pages/04.HDM Doc 2.2/14.HDM reset/default.md

```diff
@@ -44,9 +44,11 @@ Steps
 
 1. To access the HDM dashboard, select _HDM_ from _Shortcuts_ or from the Menu in the on-premises vCenter.
 2. Select the _Administration_ tab,** **then _HDM Health_ and** _HDM Reset_** (figure 43). If the _HDM Reset_ button is not enabled, then reboot the HDM appliance and try again.
-3. Select the _HDM RESET_ button to trigger the reset.
-4. You can view the task in the on-premises vCenter (figure 44).
-5. Restart the HDM appliance from vCenter and wait for the restart to complete.
+3. For HDM reset to start, administrators will have to restart the praapa IOFilter service on all ESXi of the cluster on-prem and then reboot the appliance manually. This step is applicable on all resource allocations except Ultra Lite.
+4. SPBM should be removed on VMs or VMs with SPBM should be powered off before the IOFilter restart.
+5. Select the _HDM RESET_ button to trigger the reset.
+6. You can view the task in the on-premises vCenter (figure 44).
+7. Restart the HDM appliance from vCenter and wait for the restart to complete.
```

Comment:
The ordering of the steps has to be changed. Currently it indicates to perform appliance reboot twice - Step numbered 3 & Step numbered 7.
Also if we follow these steps, we are rebooting appliance before triggering HDM reset. If the intent here is to explain the working of HDM reset, then it has to under the main section of HDM reset.
In steps, we should just have the actions that have to be performed.

Also, these are steps for HDM reset, then it would be appropriate to put Step numbered 4 in Pre-requisites sub-section, with the rider for Ultra Lite resource allocation.
Or else at least we should have this line before the Step numbered 3. We are instructing the IOFilter restart in Step numbered 3 but mentioning its pre-condition after that in Step numbered 4.

Reply:
Resolved...